app/views/menuPerDayPerPerson.scala.html

```diff
@@ -32,14 +32,14 @@
             </tr>
           </thead>
           @for(index <- listOfMenusPerDayPerPerson.indices) {
-            <tr>
+            <tr class="schedule-row">
               <td class="list-select-all">
                 <input type="checkbox" class="schedule"
                 value="@listOfMenusPerDayPerPerson(index).menuPerDayUuid"
-                name="menuPerDayUuid[@index]"
+                name="menuPerDayUuid[@index]" data-date="@listOfMenusPerDayPerPerson(index).menuDate"
                 @if(listOfMenusPerDayPerPerson(index).isSelected) { checked } >
               </td>
-              <td><label class="font-normal">@listOfMenusPerDayPerPerson(index).menuDate</label></td>
+              <td id="menuDate" value="@listOfMenusPerDayPerPerson(index).menuDate"><label class="font-normal">@listOfMenusPerDayPerPerson(index).menuDate</label></td>
```

Comment:
IDs should be unique, and since there are going to be multiple cells it's going to be a problem. Change it to a class, something like `js-menu-date`.